packages/apps/autorest/entrypoints/app.js

```diff
@@ -46,17 +47,7 @@ if (
       process.exit(1);
     }
 
-    if (v[0] > 14) {
```

Comment:
Should we still leave in this version check?

Reply:
hhm yeah, for some reason thought this was an outdated check